_posts/en/newsletters/2025-11-28-newsletter.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 'Bitcoin Optech Newsletter #382'
+permalink: /en/newsletters/2025/11/28/
+name: 2025-11-28-newsletter
+slug: 2025-11-28-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+
+FIXME:bitschmidty
+
+## News
+
+- **Stats on compact block reconstructions updates:** 0xB10C posted to
+  [Delving Bitcoin][0xb10c delving] about his stats on compact block
+  reconstruction. This is an update to a previous [optech
+  post][compact block optech], since then 0xB10C has provided three updates
+  on compact block reconstruction. He has also tracking the average size of
+  requested transactions in kB, because of the [previous
+  post][david delving post] by Gumberg. The new chart is similar looking as
```

Comment:
```suggestion
  requested transactions in kB, based on [previous
  feedback][david delving post] by David Gumberg. The new chart is similar looking as
```

Reply:
updated in [07a3d06](https://github.com/bitcoinops/bitcoinops.github.io/pull/2564/commits/07a3d066eabdb542e2935d4bec47ea87ff128811)